lib/systems/parse.nix

```diff
@@ -327,6 +327,7 @@ rec {
         }
       ];
     };
+    gnuabi64     = { abi = "64"; };
```

Comment:
This is just for mips right?

Reply:
What is the `abi = "64"` for?

Reply:
> This is just for mips right?

Yes.

> What is the abi = "64" for?

It will be passed to gcc configuration `--with-abi=` to specify (default) target abi.
I tested gcc and it seems necessary, or gcc will produce 32bit target by default, even though `mips64` is specified.

Well, I'm not sure if we should put this here, or manually specified in `crossSystem.platform.platform.gcc` when importing `<nixpkgs>` if needed.